xml_converter/generators/code_generator.py

```diff
@@ -289,12 +303,14 @@ class AttributeVariable:
     xml_fields: List[str]
     protobuf_field: str
     args: List[str] = field(default_factory=list)
-    default_xml_fields: List[str] = field(default_factory=list)
-    xml_export: str = ""
+    default_xml_field: str = ""
```

Comment:
Is this needed, or will this always be `xml_fields[0]`.

Reply:
See Issue #145